packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -2918,14 +2918,7 @@ describe('<Autocomplete />', () => {
           renderInput={(params) => <TextField {...params} autoFocus />}
         />,
       );
-      expect(handleHighlightChange.callCount).to.equal(
-        // FIXME: highlighted index implementation should be implemented using React not the DOM.
-        reactMajor > 18 ? 2 : 1,
-      );
-      expect(handleHighlightChange.args[0]).to.deep.equal([undefined, options[0], 'auto']);
-      if (reactMajor > 18) {
-        expect(handleHighlightChange.args[1]).to.deep.equal([undefined, options[0], 'auto']);
-      }
+      expect(handleHighlightChange.callCount).to.equal(0);
```

Comment:
Did you run these tests with React 18 to check that the major version check is no longer required?

Reply:
I hadn't, but I tested it now with React v18.3.1 using `pnpm use-react-version 18.3.1`, and the tests pass. Is that the right way to confirm?

Reply:
It is 👍🏼